File: Java Script/Day Four/Perfect.js
function Perfect() {

const list = [4, 9, 25, 49, 12, 16];
let isSquare = false;



for (let count = 0; count< list.length; count++) {
let number = list[count];

let i = 1;

while(i * i <= number){

if(i * i === number){
	isSquare = true;
}
else {
     isSquare = false;

i++;

}
console.log(number + " - " + isSquare);

}
}
}

File: Java/Day Four/Perfect.java
public class Perfect {

	  public int totalNumberInTheArray() {

	return 0;
}
	public String addWord(String Word) {

	return "Invalid Input";
}
	public int addNumber() {

	return 0;
}

	public static void perfectsquare() {
	
    int count = 2;

    boolean isSquare = false;
    
    int[] number = {4,8,9,12,16,25,49};
    
    for (int num = 0; num < number.length; num++) { 
	    int result = number[num];
   
    count = number[num] / count;
    isSquare = false;
     
    if (count * count == number[num]) 
    isSquare = true; 

	count++;
    
   	return perfectsquare();  
   }

}

}